src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/EnumConstantWriterImpl.java

```diff
@@ -73,24 +73,24 @@ public void addSummary(Content summariesList, Content content) {
     }
 
     @Override
-    public Content getEnumConstantsDetailsTreeHeader(TypeElement typeElement,
-            Content memberDetailsTree) {
-        memberDetailsTree.add(MarkerComments.START_OF_ENUM_CONSTANT_DETAILS);
-        Content enumConstantsDetailsTree = new ContentBuilder();
+    public Content getEnumConstantsDetailsHeader(TypeElement typeElement,
+                                                 Content memberDetails) {
+        memberDetails.add(MarkerComments.START_OF_ENUM_CONSTANT_DETAILS);
+        Content enumConstantsDetailsContent = new ContentBuilder();
```

Comment:
This is one of those cases where a short, more generic name like `result` might work better